.github/workflows/cargo-update.yml

```
@@ -21,12 +17,26 @@ jobs:
       - name: Cargo update
         run: cargo update
 
+      - name: Commit changes
+        id: commit
+        continue-on-error: true
+        run: |
+          git config user.name 'Phylum Bot'
+          git config user.email 'phylum-bot@users.noreply.github.com'
+          git commit -a -m "build: Bump Cargo.lock dependencies"
+          git push --force origin HEAD:auto-cargo-update
+
       - name: Create Pull Request
-        uses: peter-evans/create-pull-request@v4
+        if: ${{ steps.commit.outcome == 'success' }}
+        uses: actions/github-script@v6
         with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-          commit-message: "build: Bump Cargo.lock dependencies"
-          title: "build: Bump Cargo.lock dependencies"
-          body: |
-            Bump dependencies in Cargo.lock for all SemVer-compatible updates.
-          branch: auto-cargo-update
+          github-token: ${{ secrets.GH_RELEASE_PAT }}
+          script: |
+            github.rest.pulls.create({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              head: "auto-cargo-update",
+              base: context.ref,
+              title: "build: Bump Cargo.lock dependencies",
+              body: "Bump dependencies in Cargo.lock for all SemVer-compatible updates.",
+            });
```

```suggestion
            })
```
works with the semi-colon, but not necessary.

---

I'd encourage using a semicolon, even if not necessary. That way adding a new line doesn't require changing the previous one.

---

I'll preface by saying that I didn't actually *try* any of this (with or without semicolons) so there is a chance this is just bad advice...but the comment originated from reviewing the examples on the [`actions/github-script` documentation](https://github.com/actions/github-script). In all but one case there, semicolons are not used for multiline examples.

That said, this conversation will be resolved as it doesn't cause any trouble to use the semicolon and there are no other changes to make in this review.